### PR TITLE
feat: add button type

### PR DESCRIPTION
**Pull request by y-lohse**

### CODE REVIEW

**Comment by ptbrowne** [comment]
Please explain ?

**Comment by y-lohse** [comment]
Yeah sorry I should have copied the explanation in the github description 🙇

File: react/Button/Readme.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ Both look exactly the same, they share the same `theme`, `className` & `onClick`
 
 - `busy` that adds a spinner (default `false`)
 - `disabled` that disable the click event (default `false`)
+- `type`, eg. `submit` or `reset` (default `submit`)
```
Comment (GoOz) [inline react/Button/Readme.md]: Could also be `button` but I don't know if you wanted to list them all or if this was just for the example.

Comment (y-lohse) [reply react/Button/Readme.md]: Yeah no it was just to list a few :) [Related](http://theoatmeal.com/comics/ie) 😛

**Pull request merged**